src/org/newdawn/slick/GameContainer.java

```
@@ -843,7 +843,11 @@ public void setTargetFrameRate(int fps) {
 	 */
 	public void setVSync(boolean vsync) {
 		this.vsync = vsync;
-		Display.setVSyncEnabled(vsync);
+		try{
+			Display.setVSyncEnabled(vsync);
+		} catch (Exception e) {
+			Log.error("GL error", e);
```

Why are you suppressing these errors?